### PR TITLE
Update UG for addPatient command

### PR DESCRIPTION
**Pull request by chiamyunqing**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,19 +67,19 @@ Shows a message explaning how to access the help page.
...
 
-Adds a person to the address book.
+Adds a quarantined individual to the application.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `addpatient n/NAME t/TEMPERATURE d/PERIOD_OF_STAY p/PHONE_NUMBER a/AGE [c/COMMENT]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+The comment field for the quarantined individual is optional.
```
Comment (LeeMingDe) [inline docs/UserGuide.md]: do you think we should add in details like the temp must be 1dp or that the age must be in the range from 0-120?

**Review by LeeMingDe — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (chiamyunqing) [reply docs/UserGuide.md]: I think the temp should write but the age probably no need haha.

**Review by w-yeehong — APPROVED** [review]
LGTM

**Pull request merged**